src/Sylius/Bundle/ShopBundle/Resources/config/services/listeners.xml

```diff
@@ -52,5 +52,11 @@
             <argument type="service" id="sylius.context.locale" />
             <tag name="kernel.event_listener" event="sylius.order.pre_complete" method="assignLocale" />
         </service>
+
+        <service id="sylius.listener.session_cart" class="Sylius\Bundle\ShopBundle\EventListener\SessionCartSubscriber">
+            <argument type="service" id="sylius.context.cart" />
+            <argument>_sylius.cart</argument>
```

Comment:
After further investigation, we use it in three services definitions and `ShopUserLogoutHandler` has it hardcoded, we can extract it to some parameter in another PR.